HowAwayAreYou/Domain/ImageProcessor.swift

```diff
@@ -15,6 +15,7 @@ protocol ImageProcessorInput: AnyObject {
     // For some reason (I guess it's something related to the frame rate) if you don't ask the CameraIO for metadata objects like faces,
     // CMSampleBufferGetImageBuffer will fail to get the image buffer from the output of AVCaptureDataOutputSynchronizer.
     // It's really weird that even I collect the metadata objects in CameraIO and just ignore it on publisher map, the problem still occurs.
+    // TODO: ↑ Maybe this is now solved on iOS 14. Should try later.
```

Comment:
<!--
  0 failure: 
  1 warning:  TODOs should be r...
  
  
  DangerID: danger-id-Danger;
  File: HowAwayAreYou/Domain/ImageProcessor.swift;
  Line: 18;
-->

- :warning: TODOs should be resolved (↑ Maybe this is now solved on ...). (`todo`)